code/API_definitions/NetworkSliceBooking.yaml

```diff
@@ -0,0 +1,293 @@
+                
+  /sessions/{sessionId}:
+    delete:
+      tags:
+        - NSB Sessions
+      summary: Delete a NSB session
+      description: Deleting NSB session
+      operationId: deleteSession
+      parameters:
+        - name: sessionId
+          in: path
+          description: Session ID that was obtained from the createSession operation
+          required: true
+          schema:
+            $ref: "#/components/schemas/SessionId"
+      responses:
+        "204":
+          description: Session deleted
+    get:
+      tags:
+        - NSB Sessions
+      summary: Get NSB session information
+      description: Querying for NSB session resource information details
+      operationId: getSession
+      parameters:
+        - name: sessionId
+          in: path
+          description: Session ID that was obtained from the createSession operation
+          required: true
+          schema:
+            $ref: "#/components/schemas/SessionId"
+      responses:
+        "200":
+          description: Contains information about active session
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SessionInfo"
+components:
+  schemas:
+    SessionId:
+      description: Session ID in UUID format
+      type: string
+      format: uuid
+
+    SessionInfo:
+      description: Session related information.
+      allOf:
+        - $ref: "#/components/schemas/CreateSession"
+        - type: object
+          properties:
+            sessionId:
+              $ref: "#/components/schemas/SessionId"
+            result:
+              type: string
+              enum:
+                - Success
+                - Fail
+          required:
+            - result
+
+
+
+    CreateSession:
+      description: Attributes required to create a session
+      type: object
+      properties:
+        ServiceTime:
+          $ref: "#/components/schemas/ServiceTime"
+        ServiceArea:
+          $ref: "#/components/schemas/Area"
+        SLAProfile:
+          $ref: "#/components/schemas/SLAProfile"
+      required:
+        - ServiceTime
+        - ServiceArea
+        - SLAProfile
+
+    SLAProfile:
+      type: object
+      properties:
+        MaxNumofTerminals:
+          $ref: "#/components/schemas/NumofTerminals"
+        DLThroughputPerTerminal:
+          $ref: "#/components/schemas/Throughput"
+        ULThroughputPerTerminal:
+          $ref: "#/components/schemas/Throughput"
+        DLLatency:
+          $ref: "#/components/schemas/Latency"
+        ULLatency:
+          $ref: "#/components/schemas/Latency"
+
+
+    Latency:
+      type: integer
+      example: 10
+      description: |
+        An attribute specifies the required DL packet transmission latency (millisecond) 
+        through the RAN, CN, and TN part of 5G network.
+      format: int32
+
+
+    NumofTerminals:
+      type: integer
+      example: 5
+      description: Number of terminals

```

Comment:
Add some descriptions.

Reply:
"Maximum Num of Terminals" is suggested to keep it. This is an attribute that used to evaluation whether Network can assure the slice to be booked and meet the customers' need.